pkgs/applications/graphics/freecad/default.nix

```diff
@@ -1,48 +1,114 @@
-{ lib, mkDerivation, fetchFromGitHub, fetchpatch, cmake, ninja, coin3d,
-xercesc, ode, eigen, qtbase, qttools, qtwebengine, qtxmlpatterns, wrapQtAppsHook,
-opencascade-occt, gts, hdf5, vtk, medfile, zlib, python3Packages, swig,
-gfortran, libXmu, soqt, libf2c, libGLU, makeWrapper, pkg-config, mpi,
-spaceNavSupport ? true, libspnav, qtx11extras }:
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, ninja
+, GitPython
+, boost
+, coin3d
+, eigen
+, gfortran
+, gts
+, hdf5
+, libGLU
+, libXmu
+, libf2c
+, libspnav
+, matplotlib
+, medfile
+, mpi
+, ode
+, opencascade-occt
+, pivy
+, pkg-config
+, pycollada
+, pyside2
+, pyside2-tools
+, python
+, python3Packages
```

Comment:
I'm not enthuastic about having the interpreter `python`, the package set `python3Packages`, *and* the individual python packages `pyside2`, .., as arguments.  Personally I would prefer taking `python3` als an argument, and then use `with python3.pkgs; ...` in the derivation.

Reply:
Ups, little mistake. I was reformatting the expression and forgot this.

Reply:
P.S.: is it expected that it does not build locally in my machine? It was hanging...

Reply:
On the other hand, I prefer explicit dependencies instead of using `with pythonPackages`.